testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -660,6 +661,41 @@ public void testCommentViewColumn()
         }
     }
 
+    @Test
+    public void testCommentMaterializedViewColumn()
+    {
+        if (!hasBehavior(SUPPORTS_COMMENT_ON_MATERIALIZED_VIEW)) {
+            if (hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW)) {
+                String viewName = "test_materialized_view_" + randomNameSuffix();
+                assertUpdate("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM nation");
+                assertQueryFails("COMMENT ON COLUMN " + viewName + ".regionkey IS 'new region key comment'", "This connector does not support setting materialized view column comments");
+                return;
+            }
+            throw new SkipException("Skipping as connector does not support MATERIALIZED VIEW COLUMN COMMENT");
+        }
+
+        String viewName = "test_materialized_view_" + randomNameSuffix();
+        assertUpdate("CREATE MATERIALIZED VIEW " + viewName + " AS SELECT * FROM nation");
+
+        String viewColumnName = "regionkey";
+
+        // comment set
+        assertUpdate("COMMENT ON COLUMN " + viewName + "." + viewColumnName + " IS 'new region key comment'");
+        assertThat(getColumnComment(viewName, viewColumnName)).isEqualTo("new region key comment");
+
+        // comment updated
+        assertUpdate("COMMENT ON COLUMN " + viewName + "." + viewColumnName + " IS 'updated region key comment'");
+        assertThat(getColumnComment(viewName, viewColumnName)).isEqualTo("updated region key comment");
+
+        // comment set to empty
+        assertUpdate("COMMENT ON COLUMN " + viewName + "." + viewColumnName + " IS ''");
+        assertThat(getColumnComment(viewName, viewColumnName)).isEqualTo("");
+
+        // comment deleted
+        assertUpdate("COMMENT ON COLUMN " + viewName + "." + viewColumnName + " IS NULL");
+        assertThat(getColumnComment(viewName, viewColumnName)).isEqualTo(null);
```

Comment:
It would be better to drop materialized view because this test runs on shared services. Same for the above `hasBehavior(SUPPORTS_CREATE_MATERIALIZED_VIEW)`.

Reply:
drop added